cmd/wire/main.go

```diff
@@ -115,6 +116,7 @@ func (*genCmd) Usage() string {
 }
 func (cmd *genCmd) SetFlags(f *flag.FlagSet) {
 	f.StringVar(&cmd.headerFile, "header_file", "", "path to file to insert as a header in wire_gen.go")
+	f.StringVar(&cmd.outputFileName, "output_file", "", "change the output default file 'wire_gen.go' for a custom name.")
```

Comment:
I believe you can put "wire_gen.go" as the default value for the flag here, in the 3rd argument to `StringVar`.

Then the help text would be something like "the output file name". I don't remember if `subcommands` will automatically include the default in the help text; if it doesn't, please add it.

Reply:
I'm agree with you. I'll change as son as I can

Reply:
I changed this StringVar to prefix and I left the default value as "", because It's what I understand that should be with the prefixes.

I checked and as you said @vangent, the text in the description appear in the subcommands automatically.